updates/testing.json

```diff
@@ -81,18 +81,23 @@
       "metadata": {
         "barrier": {
           "reason": "https://docs.fedoraproject.org/en-US/fedora-coreos/update-barrier-signing-keys/"
```

Comment:
I'm confused here. All three streams have two f36 update barriers with https://docs.fedoraproject.org/en-US/fedora-coreos/update-barrier-signing-keys/ for reason. I'm not sure how we got here, but only one should be necessary.

Reply:
I think it's because of the confusion about how we were handling this in the past. I think the important thing is that going forward we won't have this problem.

Reply:
Cleanup in https://github.com/coreos/fedora-coreos-streams/pull/607.